### ISSUE TITLE
Draft update for the no results message for global search

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#3603]
### Summary

**What we are after:**
Our "no results" message links the classic site, so we need to remove that reference. We also should change the contact info message to match the one we use in legal searches.

Screenshot of current message:
![image](https://user-images.githubusercontent.com/24437369/76330183-eb0f1380-62c3-11ea-99e9-63597e1e8802.png)

### Completion criteria

- [x] Draft new message
- [x] Review and approval by PMs and content team
- [x]  Create front end implementation ticket.

### Future work

- Front end ticket to be created after message drafted.

**Comment by dorothyyeager** [comment]
Screenshot of no results message in legal search...

![image](https://user-images.githubusercontent.com/24437369/76556106-4e897480-646f-11ea-9c29-c6d79b25074f.png)

**Comment by dorothyyeager** [comment]
cc: @PaulClark2 @AmyKort @patphongs @kathycarothers @JonellaCulmer - my suggested revision is below. Let me know what you think. @patphongs mentioned on Slack that there are some other pages on our site which currently link classic in a search situation. I can put in a ticket as needed to redraft or list them in the front end ticket once this message is approved, as needed.


**Draft new message for no results in global search:**

**No results**
Sorry, we didn't find any pages matching "search term."

Think this was a mistake? 
Try one of the other search tools or submit feedback
[Email our team button] [File an issue button]

**Comment by kathycarothers** [comment]
@dorothyyeager The new message looks good to me. Under "Try one of the other search tools or submit feedback" I would also include the button for [fec.gov].

**Comment by JonellaCulmer** [comment]
@dorothyyeager To which other search tools are you referring? We should make it clear where we're directing folks. 

In the Legal search example it says Try FEC.gov but it's not working currently, do you know where that's supposed to go? cc: @patphongs But you're recommending we remove that anyway, right?

**Comment by dorothyyeager** [comment]
@JonellaCulmer To answer your question, I tried it out and notice it's not working either in the legal search example. It points to http://search04.fec.gov/ and returns a time-out error.

I was thinking we should remove it, especially for global search, where the results page links all the other searches one can do under "search tools". We probably should investigate why the link isn't working on legal search. cc: @patphongs

**Comment by PaulClark2** [comment]
@patphongs isn't `search04` one of the servers that our old Vivisimo search lived on?

**Comment by dorothyyeager** [comment]
Going to close out this ticket since message is drafted, and open up two follow up tickets - one for front end to implement the no results message (minus the try fec.gov link) and one to fix the try fec.gov link on the legal search results.

**Comment by dorothyyeager** [comment]
Implementation ticket for new message opened at #3619 and fixing the legal search message at #3618 

Thanks everyone for reviewing and closing this ticket.
